core/templates/pages/teach-page/teach-page.component.ts

```diff
@@ -16,64 +16,61 @@
  * @fileoverview Component for the teach page.
  */
 
-require('base-components/base-content.directive.ts');
-require(
-  'components/common-layout-directives/common-elements/' +
-  'background-banner.component.ts');
+import { Component, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { SiteAnalyticsService } from 'services/site-analytics.service';
```

Comment:
Add a line break here. 

Angular imports, followed by line break and then oppia imports.

Reply:
Done, also fixed order